content/en/pod.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Pod
+status: Feedback Appreciated
+category: concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## What it is
+
+A pod is the smallest executable unit in [Kubernetes](/kubernetes/), representing an essential building block for deploying and managing containerized applications. 
+Each pod contains a single application instance and can hold one or more [containers](/container/).
+Kubernetes manages pods as part of a larger deployment and can scale pods [vertically](/vertical-scaling/) or [horizontally](/horizontal-scaling/) up and down.
+
+## Problem it addresses
+
+While containers generally act as independent units that run and control a certain workload, there are cases when containers need to interact and be controlled in a tightly coupled manner. 
+
+Managing these closely related containers individually would lead to management task redundancies.
+For example, the operator would have to repeatedly determine the placement of related containers to ensure they remain together, even though they already specified it during the initial setup.  
+And although the lifecycles of these related containers need to be synchronized, they can only be managed individually.  
+
+Pods bundle closely tied containers into a single unit, significantly simplifying container operations. 
+
+## How it helps
+
+Pods allow operators to group containers with dependencies and manage them as one unit. 
```

Comment:
```suggestion
Pods bundle closely tied containers into a single unit, significantly simplifying container operations.
```
replacing with the sentence from above, because sounds more neat :)